tests/components/swidget/__init__.py

```diff
@@ -0,0 +1 @@
+"""Setup for tests used for testing the Swidget integration."""
```

Comment:
While (afaik) tests for your platforms are no hard requirement atm, I would suggest you add test cases for them.

Reply:
There will be a significant test suite added in the near future.